src/forms/SelectionList/SelectionList.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect, useMemo } from 'preact/hooks';
+import PropTypes from 'prop-types';
+
+import Heading from '../../typography/Heading/Heading';
+import CSS from './styles.css';
+
+const SelectionList = ({ className, dataTestId, value, title, options, id, onChange }) => {
+	const [currentValue, setCurrentValue] = useState(null);
+	const delayCallback = 250;
```

Comment:
(naming) please rename this to: `callbackDelayInMilliseconds` and move it to a new line above line: 7.